### PR TITLE
RFC: add RFC for TiKV community re-org

### PR DESCRIPTION
**Pull request by dcalvin**
This RFC proposes an upgraded organization structure for the TiKV community, with more community roles and community organizations introduced to better stimulate the vitality of the community and maintain a positive and healthy community environment. This RFC also proposes the constitution and the operation of the Special Interest Group (SIG).

Signed-off-by: Calvin Weng <wenghao@pingcap.com>

### CODE REVIEW

File: text/2019-11-01-tikv-community-reorg-sig.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+There are two parts for the designs. Part 1 focuses on the organization changes from the perspective of the community organization; part 2 focuses on SIG operations and constitutions.
+
+### Upgraded community structure
+
+#### New roles introduced
+
+![TiKV Community Organization](../media/tikv-community-org.png)
+
+The above picture reflects the first change in this community structure upgrade - the roles of Reviewer and Active Contributor will be introduced to the developer Group. The current Developer Group includes the following:
```
Comment (gingerkidney) [inline text/2019-11-01-tikv-community-reorg-sig.md]: The new Developer Group includes the follwing:

**Comment by gingerkidney** [comment]
@siddontang @winkyao PTAL

File: text/2019-11-01-tikv-community-reorg-sig.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+In addition, this picture also reflects the relationship between community roles and SIGs.
+
+- Active Contributor
+    + Must be a Contributor with at least 8 PRs merged to the master branch within 1 year.
+    + To join an SIG, a Contributor needs to contribute more than 8 PRs to the module that the SIG is responsible for within 1 year, so that he/she can be invited to join the SIG for targeted learning and contributions.
+
+- Reviewer
+    + Must belong to a SIG; has the obligation to review PRs.
+  + Generated from Active Contributors. When an Active Contributor has significant contributions to the module and is nominated by 2 or more Committers, he/she will be invited to be a Reviewer of the module.
```
Comment (sunxiaoguang) [inline text/2019-11-01-tikv-community-reorg-sig.md]: This line has different indention on my computer.

File: text/2019-11-01-tikv-community-reorg-sig.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- Before starting a SIG, the constitution will be established in advance. The detailed steps of creating the constitution can be found on the corresponding document.
+- Except for holidays, the regular meeting is organized at least once every two weeks.
+- The assignable tasks will be graded by difficulty levels and claimed by SIG members.
+
+#### SIG Roles
+
+- **Active Contributors**
+
+    - Active contributors will be invited to the SIG
+    - Keep active inof at least one module in the SIG
```
Comment (sunxiaoguang) [inline text/2019-11-01-tikv-community-reorg-sig.md]: Sorry for my ignorant, that does inof mean exactly?

Comment (dcalvin) [reply text/2019-11-01-tikv-community-reorg-sig.md]: Sorry, a typo. Now fixed.

File: text/2019-11-01-tikv-community-reorg-sig.md (hunk 2 above)
Comment (dcalvin) [reply text/2019-11-01-tikv-community-reorg-sig.md]: Fixed.

**Comment by dcalvin** [comment]
PTAL again. @sunxiaoguang  Thanks.

**Pull request merged**